packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -495,4 +519,54 @@ export enum LaunchType {
    * The service will be launched using the FARGATE launch type
    */
   FARGATE = 'FARGATE'
-}
\ No newline at end of file
+}
+
+/**
+ * Options for adding an AWS x-ray sidecar container to a Task Definition
+ */
+export interface TracingOptions {
+  /**
+   * The minimum number of CPU units to reserve for the container.
+   *
+   * @default 256 for services running on Fargate
+   */
+  readonly cpu?: number
+
+  /**
+   * The hard limit (in MiB) of memory to present to the container.
+   *
+   * If your container attempts to exceed the allocated memory, the container
+   * is terminated.
+   *
+   * At least one of memoryLimitMiB and memoryReservationMiB is required for non-Fargate services.
+   */
+  readonly memoryLimitMiB?: number;
+
+  /**
+   * The soft limit (in MiB) of memory to reserve for the container.
+   *
+   * When system memory is under contention, Docker attempts to keep the
+   * container memory within the limit. If the container requires more memory,
+   * it can consume up to the value specified by the Memory property or all of
+   * the available memory on the container instance—whichever comes first.
+   *
+   * At least one of memoryLimitMiB and memoryReservationMiB is required for non-Fargate services.
+   *
+   * @default 256
+   */
+  readonly memoryReservationMiB?: number;
+
+  /**
+   * Specifies whether or not this container is essential.
+   *
+   * @default false
+   */
+  readonly essential?: boolean;
+
+  /**
+   * Specifies whether or not to send logs to Cloudwatch Logs.
+   *
+   * @default true
+   */
+  readonly enableLogging?: boolean;
```

Comment:
loggingEnabled

Reply:
We call it enableLogging everywhere else, wanted to maintain consistency.